autoload/sqhell.vim

```diff
@@ -93,11 +93,24 @@ function! sqhell#GetColumnName()
     normal f|
     let end = col('.')-2
     let attr = getline('.')[start:end]
-    let attr = substitute(attr, '^\s*\(.\{-}\)\s*$', '\1', '')
+    let attr = sqhell#TrimString(attr)
     call setpos('.', savecurpos)
     return attr
 endfunction
 
+function! sqhell#GetTableHeader()
```

Comment:
I imagine this is mysql specific as it depends on the format that Database is giving back correct?
If that's the case can you move it into `autoload/mysql.vim`?

Thanks!